example-app/src/admin/resources/photo/photo.resource.ts

```diff
@@ -1,4 +1,4 @@
-import uploadFeature from '@admin-bro/upload-files'
+import uploadFeature from '@admin-bro/upload'
```

Comment:
renamed the package to @admin-bro/upload because it is the same as git repo and shorter